batch_job/src/work_queue_factory.c

```diff
@@ -863,11 +865,13 @@ static void show_help(const char *cmd)
 	printf(" %-30s Specify amazon machine image (AMI). (for use with -T amazon)\n", "--amazon-ami");
 	printf(" %-30s Wrap factory with this command prefix.\n","--wrapper");
 	printf(" %-30s Add this input file needed by the wrapper.\n","--wrapper-input");
+	printf(" %-30s Specify ip address to mesos master node (for use with -T mesos)\n", "--mesos-master");
```

Comment:
In general, ask the user for hostnames, unless there is some specific need for IP addresses.